myapp/app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,17 @@
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def log_out
+    session.delete(:user_id)
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
```suggestion
    current_user.present?
```
の方が分かりやすそうですがrubocop的に問題なかったですかね？

Reply:
ご指摘ありがとうございます！
rubocop的には問題なかったのですが、おっしゃる通り確かにわかりづらいので修正しました！

https://github.com/Fablic/training/pull/1175/commits/ae4460e6d0e8d76a0318be49c16e7bbf62050e41